### PR TITLE
Eng 9475 - deployment schema change based on Andrew's feedback

### PR DESCRIPTION
**Pull request by manjujames**

### CODE REVIEW

File: src/frontend/org/voltdb/compiler/DeploymentFileSchema.xsd [hunk 1]
```diff
@@ -335,16 +335,12 @@
...
       <xs:element name="property" type="PropertyType" minOccurs="0" maxOccurs="unbounded"/>
     </xs:sequence>
     <xs:attribute name="module" type="xs:string" />
     <xs:attribute name="type" type="ServerImportEnum" use="required" />
     <xs:attribute name="enabled" type="xs:boolean" default="true"/>
-  </xs:complexType>
-
-  <xs:complexType name="ImportTransformerType">
-    <xs:attribute name="name" type="xs:string" use="required"/>
+    <xs:attribute name="format" type="xs:string"/>
```
Comment (akhanzode) [inline src/frontend/org/voltdb/compiler/DeploymentFileSchema.xsd]: Do you want to default in xsd?

File: src/frontend/org/voltdb/utils/CatalogUtil.java [hunk 2]
```diff
@@ -1396,8 +1396,7 @@ private static void setImportInfo(Catalog catalog, ImportType importType) {
             }
 
             Properties processorProperties = checkImportProcessorConfiguration(importConfiguration);
-            String transformer = (importConfiguration.getTransformer() == null) ?
-                    null : importConfiguration.getTransformer().getName().trim();
+            String transformer = importConfiguration.getFormat();
```
Comment (akhanzode) [inline src/frontend/org/voltdb/utils/CatalogUtil.java]: can you call it formatter?

File: src/frontend/org/voltdb/utils/CatalogUtil.java [hunk 3]
```diff
@@ -1396,8 +1396,7 @@ private static void setImportInfo(Catalog catalog, ImportType importType) {
...
             if (transformer != null && !transformer.isEmpty()) {
                 processorProperties.put(ImportDataProcessor.IMPORT_TRANSFORMER, transformer);
```
Comment (akhanzode) [inline src/frontend/org/voltdb/utils/CatalogUtil.java]: Same lets not have 2 terminology formatter and transformer.

File: src/frontend/org/voltdb/compiler/DeploymentFileSchema.xsd (hunk 1 above)
Comment (manjujames) [inline src/frontend/org/voltdb/compiler/DeploymentFileSchema.xsd]: Yea, I did think about it. Then decided to think about it and do it when we do the real implementation.

**Comment by akhanzode** [comment]
Rest looks good.

File: src/frontend/org/voltdb/utils/CatalogUtil.java (hunk 3 above)
Comment (manjujames) [inline src/frontend/org/voltdb/utils/CatalogUtil.java]: OK. I was trying to minimize the number of changes because this whole code is temporary anyway..

**Comment by manjujames** [comment]
squash merged.

**Pull request closed without merging**